### PR TITLE
Add LikeCoin in sponsor section

### PR DESCRIPTION
**Pull request by howawong**

### CODE REVIEW

**Comment by howawong** [comment]
Desktop Preview:
preview

Mobile Preview:
preview_mobile

**Comment by howawong** [comment]
@vincicat  please review and give comments ASAP.

File: css/base.css [hunk 1]
```diff
@@ -923,5 +923,9 @@ textarea {
     text-align: center;
 }
 
+.sponsor-logo{
+    float: left;
+    margin-right: 10px;
+}
```
Comment (vincicat) [inline css/base.css:929]: Content related style is not supposed to be at the base stylesheet

File: css/base.css [hunk 2]
```diff
@@ -495,7 +495,7 @@ textarea {
 }
 
 .col-seven {
-    width: 58.33333%;
+    width: 30%;
 }
```
Comment (vincicat) [inline css/base.css:500]: The most powerful and most difficult of css is the "Cascading" nature. Unless necessary, prefer override but not changing the third party base css.

File: index.html [hunk 3]
```diff
@@ -422,11 +422,14 @@ <h3 class="h4">Venue Sponsor</h3>
...
 
                     <h3 class="h4">Sponsor</h3>
-
                     <div>
-                        <a href="http://www.acrossasia.hk"><img src="images/across_asia.png" alt="Across Asia"></a>
+                        <div class="sponsor-logo">
+                            <a href="http://www.acrossasia.hk"><img src="images/across_asia.png" alt="Across Asia"></a>
+                        </div>
+                        <div class="sponsor-logo">
+                            <a href="https://like.co"><img src="images/likecoin_logo.png" alt="Across Asia"></a>
```
Comment (vincicat) [inline index.html:430]: 1. Actually there is much easier solution here: `<a>` is an inline element, it ALREADY do the thing that requires float to do, without the side effects. 
And it is how "Organizers" 's logo works. 
2. alternative: re-use the grid system the template provided.
3. alt-text is wrong.

File: css/base.css (hunk 2 above)
Comment (vincicat) [reply css/base.css:500]: All the grid systems (Bootstrap, Foundation etc) suppose you are going to change the **class name in HTML** but **not changing the css**.
If other grid size is used the change can be catastrophic.

**Comment by vincicat** [comment]
commit 0755bfd1779583aca8e03e39b057f81b78a6b9ca provided a better solution on the issue, PR closed.

**Pull request closed without merging**